imports/pages/give/index.js

```diff
@@ -22,7 +35,7 @@ const Routes = [
   {
     path: "give",
     component: Layout,
-    indexRoute: { onEnter: (nextState: Object, replace: Function) => replace("/give/home") },
+    indexRoute: { onEnter: (nextState: Object, replace: Function) => replace(defaultRoute()) },
```

Comment:
@edolyne this won't work as expected. If you go through the history, @richarddubay implemented a variation of this once already we can pick up